Source/WebKit/WPEPlatform/wpe/wayland/WPEViewWayland.cpp

```diff
@@ -358,6 +358,11 @@ static gboolean wpeViewWaylandRenderBuffer(WPEView* view, WPEBuffer* buffer, con
     if (!wlBuffer)
         return FALSE;
 
+    if (wpe_view_get_width(view) != wpe_buffer_get_width(buffer) || wpe_view_get_height(view) != wpe_buffer_get_height(buffer)) {
```

Comment:
You can't compare view and buffer size, because the view size is in logical coords.

Reply:
Could you elaborate a bit more on the explanation? I don’t quite understand the real difference between logical coordinates and the buffer size. The empirical tests I’ve done suggest that they are indeed comparable. Aren’t both measured in _pixels_? Could it be because the window values are affected by a `scale` factor? Can you suggest an alternative?

Reply:
It works if device scale factor is 1 only because coords match in that case. The buffer size is scaled by device scale factor, while the view size isn't.

Reply:
So if  the patch has in consideration the scale factor I understand it will be right. Right?